frontend/src/Context/SessionProvider/session.tsx

```diff
@@ -229,6 +241,22 @@ const SessionProvider = ({ children }: SessionProviderProps): ReactElement => {
 
   const { user } = useUserContext();
   const [connected, setConnected] = useState(false);
+  // This ref is used to track the current captions ID for captions management
+  const currentCaptionsIdRef = useRef<string | null>(null);
+  // This ref is used to track the current room name for captions management
+  const currentRoomNameRef = useRef<string | null>(null);
+  // This ref is used to track the number of active participants using captions
+  const captionsActiveCountRef = useRef<number>(0);
+
+  useEffect(() => {
+    if (connected && !currentCaptionsIdRef.current) {
+      // Request captions status from existing participants
+      vonageVideoClient.current?.signal({
+        type: 'captions',
+        data: JSON.stringify({ action: 'request-status' }),
+      });
+    }
```

Comment:
Could we move this to the backend and use backend signaling? We should use the backend as the source of truth for new joiners.